src/main/java/com/exadel/discount/service/impl/JwtValidationServiceImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.exadel.discount.service.impl;
+
+import com.exadel.discount.entity.Role;
+import com.exadel.discount.service.JwtExtractionService;
+import com.exadel.discount.service.JwtValidationService;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.stream.Stream;
+
+import static org.apache.commons.lang3.StringUtils.contains;
+
+@Service
+@Setter
+@Slf4j
+@RequiredArgsConstructor
+public class JwtValidationServiceImpl implements JwtValidationService {
```

Comment:
Looks like redundant class and methods

Reply:
This is done locally.

Reply:
It was resolved in commit "Commit with amendments №1".